src/libmongoc/tests/test-libmongoc.c

```diff
@@ -2094,17 +2094,18 @@ _parse_server_version (const bson_t *buildinfo)
    ASSERT (bson_iter_recurse (&iter, &array_iter));
 
    /* Server returns a 4-part version like [3, 2, 0, 0], or like [3, 2, 0, -49]
-    * for an RC. Bail if number of parts is ever not 4. */
+    * for an RC. Ignore the 4th part since RCs are equivalent to non-RCs for
+    * testing purposes. */
```

Comment:
Changes to `test-libmongoc` here are to ignore prerelease versions. Once [CDRIVER-3940](https://github.com/mongodb/mongo-c-driver/pull/767) is merged, I'll rebase.